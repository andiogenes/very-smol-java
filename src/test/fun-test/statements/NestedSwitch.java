class Main {
    int x = 0;

    void nestedSwitch() {
        switch (1) {
            case 1:
                switch (x) {
                    default:
                        return;
                }
                x = 100;
        }
        x = 10;
    }

    void main() {
        nestedSwitch();
    }
}