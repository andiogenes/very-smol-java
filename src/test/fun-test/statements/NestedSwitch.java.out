x : INT = 0 : INT
